opentelemetry-sdk/tests/trace/export/test_export.py

```diff
@@ -176,7 +198,6 @@ def test_batch_span_processor_lossless(self):
 
         for _ in range(512):
             _create_start_and_end_span("foo", span_processor)
-
```

Comment:
I notice that one of the `sdk` (`TestBatchExportSpanProcessor.test_batch_span_processor_lossless`) test cases is failing. Apparently adding a small delay here makes that test pass again:

``` diff
diff --git a/opentelemetry-sdk/tests/trace/export/test_export.py b/opentelemetry-sdk/tests/trace/export/test_export.py
index 83bcd5d6..b7fc30bd 100644
--- a/opentelemetry-sdk/tests/trace/export/test_export.py
+++ b/opentelemetry-sdk/tests/trace/export/test_export.py
@@ -198,6 +198,8 @@ class TestBatchExportSpanProcessor(unittest.TestCase):
 
         for _ in range(512):
             _create_start_and_end_span("foo", span_processor)
+        from time import sleep
+        sleep(1)
         self.assertTrue(span_processor.force_flush())
         self.assertEqual(len(spans_names_list), 512)
         span_processor.shutdown()
```